src/lib/charts/ChartLegend/ChartLegend.tsx

```diff
@@ -1,25 +1,66 @@
+import { ChartType } from 'chart.js';
```

Comment:
We are going to have some merge conflicts here.

I suppose I'll need to update my legend component sooner than later!

Once you merge this in, I can work on updating to include your work.